frost-p256/src/tests.rs

```diff
@@ -1,153 +1,24 @@
+use lazy_static::lazy_static;
 use rand::thread_rng;
+use serde_json::Value;
 
 use crate::*;
 
-mod vectors;
-
-use vectors::*;
+lazy_static! {
+    pub static ref P256_SHA256: Value =
+        serde_json::from_str(include_str!("tests/vectors.json").trim())
```

Comment:
Wait, where are these p256 vectors?

Reply:
OH i'm sorry they are in `src/tests/`, not `tests/` as in 'integration' tests for the crate